helm/node-normalization-web-server/templates/ingress.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "node-normalization-webserver.fullname" . }}-web-service-trapi-1-1
+  labels:
+    {{- include "node-normalization-webserver.selectorLabels" . | nindent 4 }}
+  annotations:
+    kubernetes.io/ingress.class: "{{ .Values.ingress.class }}"
+    nginx.ingress.kubernetes.io/enable-cors: "true"
+    nginx.ingress.kubernetes.io/proxy-body-size: 256m
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+spec:
+  rules:
+    - host: {{ .Values.ingress.hostName | quote }}
+      http:
+        paths:
+          - path: "/1.1(/|$)(.*)"
+            backend:
+              serviceName: {{ include "node-normalization-webserver.fullname" . }}-web-service-trapi-1-1
+              servicePort: {{ .Values.web.port }}
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "node-normalization-webserver.fullname" . }}-web-service-trapi-1-1-2
```

Comment:
are we still using that -trapi-1-1 on everything? i am trying to make the app version, image version, and repo tag all be the same across all the projects i touch now.